mri/operators/fourier/non_cartesian.py

```diff
@@ -234,8 +234,8 @@ def __init__(self, samples, shape, n_coils=1, density_comp=None,
         )
 
     def op(self, image, interpolate_data=False):
-        """ This method calculates the masked non-cartesian Fourier transform
-        of a 2D / 3D image.
+        """ This method computes the masked non-uniform Fourier transform
```

Comment:
(D401) First line should be in imperative mood.
(D210) No whitespaces allowed surrounding docstring text
```suggestion
        """Compute the masked non-uniform Fourier transform
```